runtime/queries/yuck/highlights.scm

```diff
@@ -0,0 +1,66 @@
+(ERROR) @error
+
+(line_comment) @comment
+
+; keywords and symbols
+
+(keyword) @keyword
+(symbol) @tag
+
+; literals
+
+(bool_literal) @constant.builtin.boolean
+(num_literal) @constant.numeric
+
+; strings
+(string_interpolation
+  (string_interpolation_start) @punctuation.special
+  (string_interpolation_end) @punctuation.special) @embedded
```

Comment:
I think neovim uses `@embedded` for something but it shouldn't be necessary in the Helix queries

```suggestion
(string_interpolation
  (string_interpolation_start) @punctuation.special
  (string_interpolation_end) @punctuation.special)
```

Reply:
I copy/pasted this mostly from the nix highlighting queries, so it may indeed be an artifact that originated from neovim (where the nix highlights are from originally AFAIK).